src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {Breadcrumb} from 'antd';
+
+import {getParentCardArrayByIdChildren} from '../../appFunctions';
+
+export default function UserBreadcrumb(props) {
+
+    const {onClick, cardId, cards} = props;
```

Comment:
if it's not too much props you can destructure it inline (on line 7 for argument)